src/OrchardCore.Modules/OrchardCore.Security/AdminMenu.cs

```diff
@@ -17,16 +17,23 @@ public AdminMenu(IStringLocalizer<AdminMenu> localizer)
 
         public Task BuildNavigationAsync(string name, NavigationBuilder builder)
         {
-            if (String.Equals(name, "admin", StringComparison.OrdinalIgnoreCase))
+            if (!String.Equals(name, "admin", StringComparison.OrdinalIgnoreCase))
             {
-                builder.Add(S["Security"], NavigationConstants.AdminMenuSecurityPosition, security => security
-                    .AddClass("security").Id("security")
-                        .Add(S["Settings"], settings => settings
+                return Task.CompletedTask;
+            }
+
+            builder
+                .Add(S["Configuration"], configuration => configuration
+                    .Add(S["Settings"], settings => settings
+                        .Add(S["Security"], security => security.Id("security")
                             .Add(S["Security Headers"], S["Security Headers"].PrefixPosition(), headers => headers
```

Comment:
Here is the same change from https://github.com/OrchardCMS/OrchardCore/pull/11972/files

Reply:
PR #11972 was closed so the change exists only here.